source/globalCommands.py

```diff
@@ -1703,35 +1703,92 @@ def script_reportClipboardText(self,gesture):
+				pass
+			if copyMarker.compareEndPoints(copyMarker, "startToEnd") == 0:
+				# Translators: Presented when there is no text selection to copy from review cursor.
+				ui.message(_("No text to copy"))
+				api.getReviewPosition().obj._copyStartMarker = None;
+				return
+			api.getReviewPosition().obj._selectThenCopyRange = copyMarker
+			# for applications such as word, where the selected text is not automatically spoken we must monitor it ourself
+			try:
+				# old selection info must be saved so that its possible to report on the changes to the selection.
+				oldInfo=pos.obj.makeTextInfo(textInfos.POSITION_SELECTION)
+			except Exception as e:
+				log.debug("Error trying to get initial selection information %s" % e)
+				pass
+			try:
+				copyMarker.updateSelection()
+				if hasattr(pos.obj, "waitForAndSpeakSelectionChange"):
+					# wait for applications such as word to update their selection so that we can detect it
+					try:
+						pos.obj.waitForAndSpeakSelectionChange(oldInfo)
+					except Exception as e:
+						log.debug("Error trying to wait for the selection to update and then speak the selection %s" % e)
+						pass
+				# Translators: Presented when some review text has been selected
+				ui.message(_("Selection made"))
+			except NotImplementedError:
+				# we are unable to select the text, leave the _copyStartMarker in place in case the user wishes to copy the text.
+				# Translators: Presented when unable to select the marked text.
+				ui.message(_("Can't select text, press twice to copy"))
+				return
+		elif scriptHandler.getLastScriptRepeatCount()==1: # the second call, try to copy the text
+			copyMarker = pos.obj._selectThenCopyRange
+			if copyMarker.copyToClipboard():
+				# Translators: Presented when some review text has been copied to clipboard.
+				ui.message(_("Review selection copied to clipboard"))
+			else:
+				# Translators: Presented when unable to copy to the clipboard because of an error.
+				ui.message(_("Unable to copy"))
+			# on the second call always clean up the start marker
+			api.getReviewPosition().obj._selectThenCopyRange = None
+			api.getReviewPosition().obj._copyStartMarker = None
+		else: # an unknown number of getLastScriptRepeatCount() calls
 			return
-		self._copyStartMarker = None
-	# Translators: Input help mode message for copy selected review cursor text to clipboard command.
-	script_review_copy.__doc__ = _("Retrieves the text from the previously set start marker up to and including the current position of the review cursor and copies it to the clipboard")
+		return
+	# Translators: Input help mode message for the select then copy command. The select then copy command first selects the review cursor text, then copies it to the clipboard.
+	script_review_copy.__doc__ = _("On first trigger, the text from the previously set start marker up to and including the current position of the review cursor is selected. On second trigger, the text is copied to the clipboard")

```

Comment:
To be consistent with other help messages, please change "On first trigger" and "On second trigger" to "If pressed once" and "If pressed twice".